jujugui/static/gui/src/app/components/help/test-help.js

```diff
@@ -22,132 +20,7 @@ describe('Help', () => {
-                  /* eslint-enable max-len */
-                  width="24" />
-                Kubernetes tutorial
-              </a>
-            </p>
-            <p>
-              <a className="link charm-row"
-                href="https://tutorials.ubuntu.com/tutorial/get-started-hadoop-spark"
-                target="_blank">
-                <img alt="Hadoop Spark logo"
-                  /* eslint-disable max-len */
-                  src="https://api.jujucharms.com/charmstore/v5/xenial/hadoop-client-8/icon.svg"
-                  /* eslint-enable max-len */
-                  width="24" />
-                Hadoop Spark tutorial
-              </a>
-            </p>
-          </VanillaCard>
-          <VanillaCard title="Take a tour">
-            <p>
-              <img className="help__tour-image"
-                src="/static/gui/build/app/assets/images/non-sprites/tour/help@1x.png"
-                /* eslint-disable max-len */
-                srcSet="/static/gui/build/app/assets/images/non-sprites/tour/help@2x.png 2x"
-                /* eslint-enable max-len */
-                width="201" />
-              <span className="link"
-                onClick={wrapper.find('.link').at(3).prop('onClick')}
-                role="button">
-                Learn how to use the canvas.
-              </span>
-            </p>
-          </VanillaCard>
-        </div>
-        <footer className="help__footer clearfix">
-          <div className="four-col">
-            <p>
-              <span className="link"
-                onClick={wrapper.find('.link').at(4).prop('onClick')}
-                role="button">
-                Keyboard shortcuts
-              </span>
-            </p>
-            <p>
-              <a className="link"
-                href="https://jujucharms.com/how-it-works#frequently-asked-questions"
-                target="_blank">
-                FAQs
-              </a>
-            </p>
-            <p>
-              <a className="link"
-                href="https://jujucharms.com/support"
-                target="_blank">
-                Get support
-              </a>
-            </p>
-          </div>
-          <div className="four-col last-col">
-            <p>IRC channels on Freenode</p>
-            <p>
-              <a className="link external"
-                href="http://webchat.freenode.net/?channels=%23juju"
-                target="_blank">
-                #juju
-              </a>
-            </p>
-            <p>Mailing lists</p>
-            <p>
-              <a className="link external"
-                href="https://lists.ubuntu.com/mailman/listinfo/juju"
-                target="_blank">
-                The Juju project
-              </a>
-            </p>
-          </div>
-        </footer>
-      </div>
-    );
-    assert.compareJSX(wrapper.find('.help__body'), expected);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
I've updated this one component as an example.

Reply:
Nice

Reply:
💥🙌🏻💪🏻

Reply:
😆